index.html

```diff
@@ -48,7 +48,7 @@ <h1>Charlie Chrisman</h1>
       </header>
       <!-- Footer -->
       <footer id="footer">
-        <span class="copyright">&copy; Charlie Chrisman</span>
+        <span class="copyright">&copy; 2018 Charlie Chrisman</span>

```

Comment:
should this be some JS function to grab current year?

Reply:
after further review this idea has been put in the backlog. changes as is to be merged for now.